### ISSUE TITLE
Consider releasing as multiple separate modules

### ISSUE THREAD

**Issue by tomekwi** [jussi-kalliokoski/trine#47]
I used to think of the `trine/boolean/and` way of distributing modules as bullseye. Super simple, built into the language.

But I’ve recently had a very interesting conversation which led me to change my mind: <https://github.com/mattdesl/module-best-practices/pull/2>. The gist of it is “this breaks things”. It might convince you as well.

**Comment by jussi-kalliokoski** [comment]
Yes, this is actually something I've been intending to do - Trine was built with this in mind, however, for quick access one main module is also handy, like with lodash - this allows for optimizing for footprint later on in the development process instead of paying the discovery price upfront (also allows for building better editor autocompletion). The build process already has a separate `main` folder that gets published, all that's left to do is to add the source transforms and dependency mappings for the separate modules to make this work. I haven't had the time to do this myself yet, and would also be more than happy to take PRs for this. :)

**Comment by gajus** [comment]
Is there still active development going on here or has this package been superseded by something else?
